tests/cdi/cdi_test.go

```diff
@@ -2,51 +2,101 @@ package cdi_test
 
 import (
 	"os"
+	"time"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/util/rand"
 	"kubevirt.io/kubevirt-ansible/tests"
+	ktests "kubevirt.io/kubevirt/tests"
 )
 
-var _ = Describe("Importing and starting a VM using CDI", func() {
-	var dstPVCFilePath, dstVMFilePath, newPVCName, url string
+var _ = Describe("Importing and starting a VMI using CDI", func() {
+	prepareCDIResource := func(manifest, url string) string {
+		dstFilePath := tmpTestDir + "test-prepare-file-" + rand.String(5) + ".json"
```

Comment:
consider using the `filepath` pkg and calling Join to build the pathname. This is more portable across different OSs